website/content/commands/connect/envoy.mdx

```diff
@@ -227,6 +231,29 @@ This example assumes that the correct [environment variables](#api-options) are
 used to set the local agent connection information and ACL token, or that the
 agent is using all-default configuration.
 
+### Sidecar Proxy with ACLs Enabled
+
+In secure deployments, Consul's ACL system is enabled with a default `deny` policy.
+Nothing can access Consul API resources unless it presents a Consul ACL token with the
+necessary privileges. A sidecar proxy must have a Consul ACL token to act as itself and
+the service it fronts, and to access all that service's potential upstreams.
+The best way to generate such a token is to
+[use a service identity](/consul/commands/acl/token/create#create-a-token-for-a-service).
```

Comment:
@rboyer : In the future, we'll want to similarly clarify how to launch the various gateways with a token with the necessary ACL permissions. This pairs with the "write a doc on how to create the ACL token needed for various things" task.